tools/ci/docker/ydoc-server/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM node:lts-bookworm-slim
```

Comment:
We likely want to keep node version pinned here. Currently we define it in `.node-version` to `20.11.1`, so that would be the ideal choice. Though any further node-20 should work without issues as well. LTS tag will eventually change to something else.

Reply:
Agreed